### PR TITLE
Display registration date on affiliate edit page

### LINKED ISSUE #433: Show an affiliate's registration date and other fields on edit affiliate screen
A user on Twitter has suggested that the date an affiliate registered, and other registration fields should be visible on the edit affiliate screen.

Registration date will be handy. Should we add the website URL and the promotion method to the edit screen?

### PR DESCRIPTION
**Pull request by ghost1227**
Fixes #433

### CODE REVIEW

File: includes/admin/affiliates/edit.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
 				</th>
 
 				<td>
-					<input class="small-text" type="text" name="user_id" id="user_id" value="<?php echo esc_attr( $affiliate->user_id ); ?>" disabled="1" />
+					<input class="small-text" type="text" name="user_id" id="user_id" value="<?php echo date_i18n( get_option( 'date_format' ), strtotime( $affiliate->user_id ) ); ?>" disabled="1" />
```
Comment (pippinsplugins) [inline includes/admin/affiliates/edit.php]: Why are you converting user ID to a date?

**Comment by pippinsplugins** [comment]
@ghost1227 ping

**Comment by sumobi** [comment]
@ghost1227 As per @pippinsplugins's comment, could you remove the change to the user ID field where it's being converted to a date? Everything else looks good.

User ID is being converted to date:
screen shot 2016-03-13 at 5 46 51 pm

**Comment by ramiabraham** [comment]
Closing in favor of https://github.com/AffiliateWP/AffiliateWP/pull/1254

**Pull request closed without merging**